src/app/api/comments/route.ts

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+import { PrismaClient } from "@prisma/client";
+import { NextResponse, type NextRequest } from "next/server";
+import { createCommentSchema } from "./schema";
+
+const prisma = new PrismaClient();
+
+export async function POST(req: NextRequest) {
```

Comment:
is it possible to create a method for replies?  it seems there might have originally been something here for adding a reply, but it was commented out

Reply:
added an api for adding a reply